Adding HPC OnDemand help pages

milljm opened the pull request:
This is a WIP.

Closes #15299

Commit:
Adding HPC OnDemand help pages

This is a WIP.

Closes #15299

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit c7ff589edbb84aca6b140c463e38c941b37f4e2f
+Subproject commit 9ef4629999c8a5b394064786f47627dc7a407395
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/553512/) on 6f07785 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/15506/site)

This comment will be updated on new commits.

Commit:
Improve Conda Instructions/troubleshooting

Folks are missing the `conda init` step. The problem is, we can't just
tell folks exactly what to run here. They need to understand to follow
the instructions on the screen at this point (and not ours).

Commit:
Adjusting GCC minimum version

We bumped our minimum version to 5.1.

Commit:
Adding a list of libraries MOOSE uses

I was tasked with adding the libraries in use by the MOOSE Framework.

Commit:
Put it all together

Still a W.I.P.

aeslaughter commented on modules/doc/content/help/inl/hpc_ondemand.md:
```diff
@@ -0,0 +1,25 @@
+ shell prompt is obtained, by clicking on 'Clusters' and selecting any of the desired HPC machines. Each machine is reachable by the other via SSH. The terminal window is text based, making it very responsive, and simple to cut and paste command results.
+
+
+## Linux Desktop
+
+WIP
+
+## Jobs
+
+WIP
```
```suggestion
```

Commit:
Add more cool documentation

Refs #15299

Commit:
Remove Conda update instructions

Remove the bit about keeping Conda up-to-date. This is being done (and needs
to be done) during the main 'Getting Started' page, when we go to mention
_both_ Conda and MOOSE repository. They need to be done simultaneously, and
it doesn't make sense to teach users how to do that during a Conda install,
before they even have a moose repository to update.
